blazor/diagram/connectors/segments/bezier/bezier.md

```diff
@@ -0,0 +1,244 @@
+
+By default, when there are no segments defined for a bezier connector, the bezier segments will be created automatically and routed in such a way that avoids overlapping with the source and target nodes.
+
+```cshtml
+@using Syncfusion.Blazor.Diagram
+
+<SfDiagramComponent Width="1000px" Height="500px" Nodes="@nodes" Connectors="@connectors"></SfDiagramComponent>
+
+@code {
+    //Define the diagram's connector collection.
+    DiagramObjectCollection<Connector> connectors = new DiagramObjectCollection<Connector>();
+    //Define the diagram's node collection.
+    DiagramObjectCollection<Node> nodes = new DiagramObjectCollection<Node>();
+
+    protected override void OnInitialized()
+    {
+        nodes.Add(
+            new Node()
+                {
+                    ID = "node1",
+                    OffsetX = 300,
+                    OffsetY = 100,
+                    Width = 100,
+                    Height = 100,
+                    Ports = new DiagramObjectCollection<PointPort>()
+                    {
+                    new PointPort()
+                    {
+                        ID="Port1",
+                        Visibility = PortVisibility.Visible,
+                        Offset = new DiagramPoint() { X = 1, Y = 0.5 },
+                    },
+                    }
+                });
+        nodes.Add(new Node()
+            {
+                ID = "node2",
+                OffsetX = 300,
+                OffsetY = 350,
+                Width = 100,
+                Height = 100,
+                Ports = new DiagramObjectCollection<PointPort>()
+                {
+                    new PointPort()
+                    {
+                        ID="Port1",
+                        Visibility = PortVisibility.Visible,
+                        Offset = new DiagramPoint() { X = 0, Y = 0.5 },
+                    },
+                }
+            });
+        Connector connector1 = new Connector()
+            {
+                ID = "connector1",
+                SourceID = "node1",
+                TargetID = "node2",
+                SourcePortID = "Port1",
+                TargetPortID = "Port1",
+                Type = ConnectorSegmentType.Bezier,
+                Constraints = ConnectorConstraints.Default | ConnectorConstraints.DragSegmentThumb
+            };
+        connectors.Add(connector1);
+    }
+}
+```
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/Connectors/Segments).
+
+Also, if you provide segments on initial rendering, the segments collection will be updated dynamically when you move the connector ends. For static segments collection, the [BezierConnectorSettings.AllowSegmentsReset](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.BezierConnectorSettings.html#Syncfusion_Blazor_Diagram_BezierConnectorSettings_AllowSegmentsReset) property of the Connector class should be set to false.
```

Comment:
Also, if you provide segments on initial rendering, the segment collection will be updated dynamically when you move the connector ends. For static segments collection, the [BezierConnectorSettings.AllowSegmentsReset](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.BezierConnectorSettings.html#Syncfusion_Blazor_Diagram_BezierConnectorSettings_AllowSegmentsReset) property of the Connector class should be set to false.